### PR TITLE
generate css files in build to use with csp

### PR DESCRIPTION
**Pull request by nightwing**
see https://github.com/ajaxorg/ace/pull/4554 for context

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Makefile.dryice.js [hunk 1]
```diff
@@ -447,13 +449,113 @@ function buildAce(options, callback) {
...
             buildTypes();
+        extractCss(options);
             
         if (callback) 
             return callback();
         console.log("Finished building " + getTargetDir(options));
     }
 }
 
+function extractCss(options) {
```
Comment (ilya-babanov) [inline Makefile.dryice.js:460]: Is that ok that we don't await on creation of theme files here, and on `extractCss` in general? Feels like `extractCss` should accept a callback to, but I don't really have much context here.

**Comment by dimovpetar** [comment]
Hello, are there any updates for the CSP topic?

**Comment by nightwing** [comment]
Hi, CSP changes are released in v1.4.13 today.

**Comment by dimovpetar** [comment]
Thanks a lot for your support!
I guess https://github.com/ajaxorg/ace/issues/3260 and https://github.com/ajaxorg/ace/issues/3307 now can be closed.

**Pull request merged**